examples/thermostat/thermostat-common/src/atomic-write-manager.cpp

```diff
@@ -0,0 +1,535 @@
+/*
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <atomic-write-manager.h>
+
+#include <platform/internal/CHIPDeviceLayerInternal.h>
+
+using namespace chip;
+using namespace chip::app;
+using namespace chip::app::Clusters::Globals::Structs;
+using namespace chip::app::Clusters::Thermostat;
+using namespace chip::app::Clusters::Thermostat::Attributes;
+
+using imcode = chip::Protocols::InteractionModel::Status;
```

Comment:
how about using namespace chip::Protocols::InteractionModel::Status; and the using Status everywhere. Boris had a comment on my PR that imcode was wierd :)